### PR TITLE
Updated release notes with first part of resolved issues

### PR DESCRIPTION
**Pull request by Miroslav-Anashkin**
Added known issues section

### CODE REVIEW

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 1]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
 here: https://mirantis.zendesk.com/home.
+
+
+Issues fixed in this release
+----------------------------
+
+*PRD-976 Management network not configured correctly during deployment*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  IP address is now correctly assigned to the network interface, belonging to
+  management network.
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst]: Fixed the issue with the incorrect management network configuration. The IP address is now correctly assigned to the network interface that belongs to management network.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 2]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+  
+*PRD-999 Fuel-web rabbitmq use of short names should use IP addresses*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Name resolving is added for internal OpenStack network to allow hostnames
+  usage for OpenStack components.
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst:518]: Added name resolving (or resolution?) for the internal OpenStack network. This enables the usage of hostnames for OpenStack components.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 3]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+Name resolving is added for internal OpenStack network to allow hostnames
+  usage for OpenStack components.
+  
+*PRD-1507 Metadata issue in FuelWeb 3.0.1*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Kernel namespaces are enabled for Centos in order to get Neuton metadata
+  operable. Ubuntu 12.04 LTS has kernel namespaces support out of the box.
+  RedHat is going to enable kernel namespaces in ipcoming RHEL 6.5 release.
+  Updated Cirros image, included to Fuel ISO with fixed cloudinit.
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst:525]: Kernel namespaces are enabled for Centos in order to make Neuton metadata
operable. Ubuntu 12.04 LTS supports kernel namespaces support out of the box.
RedHat will enable kernel namespaces in upcoming RHEL 6.5 release.
Updated Cirros image is included to the Fuel ISO with fixed `cloudinit` - I thought you forgot spaces between "cloud in it" =)

Overall, it's unclear what "metadata issues" has actually been fixed. You describe some updates and additions, but don't tell the story about them.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 4]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+  Kernel namespaces are enabled for Centos in order to get Neuton metadata
+  operable. Ubuntu 12.04 LTS has kernel namespaces support out of the box.
+  RedHat is going to enable kernel namespaces in ipcoming RHEL 6.5 release.
+  Updated Cirros image, included to Fuel ISO with fixed cloudinit.
+  
+*PRD-1024 FuelWeb doesn't work when DHCP interface is not eth0*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Ability to set and customize DHCP interface settings was added to Fuel master 
+  node installation sequence.
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst:530]: Added the ability to set and configure DHCP settings to the Fuel Master Node installation wizard.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 5]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+  node installation sequence.
+  
+*PRD-1404 Add links to sub-headers in the Understanding the Puppet Manifest documentation page*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Added missing links to Fuel online documentation.
+  
+*PRD-1117 Mistakes on page Configuring the network of Fuel Web docs*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Fuel online docs were corrected.
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst]: FIxed Fuel online documentation

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 6]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+  
+*PRD-1117 Mistakes on page Configuring the network of Fuel Web docs*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Fuel online docs were corrected.
+  
+*PRD-1131 Glance cache must be cleaned up periodically*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  New controller health-check logic was implemented to maintain the remained
+  free space, cleanups and so on.
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst]: Added new controller health-check logic to maintain (manage?) free space, cleanups, and so on.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 7]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+  New controller health-check logic was implemented to maintain the remained
+  free space, cleanups and so on.
+  
+*PRD-1308 Need validation of network ranges* **(M/A::marked as unresolved)**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Network ranges validation was improved to block even more potentially 
+  not-working network configurations. Fuel 3.2 strongly prohibits every untagged
+  traffic over the admin (PXE) network. Please expect not all custom network 
+  settings that worked in 3.1 should continue working in 3.2!
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst]: Improved network ranges validation to block more potentially not-working network configurations. 
Fuel 3.2 strongly prohibits all untagged traffic over the admin (PXE) network. Some of custom 
network settings for 3.1 may not work in 3.2!

This is actually Enhancement, not an issue.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 8]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+  Network ranges validation was improved to block even more potentially 
+  not-working network configurations. Fuel 3.2 strongly prohibits every untagged
+  traffic over the admin (PXE) network. Please expect not all custom network 
+  settings that worked in 3.1 should continue working in 3.2!
+  
+*PRD-1379 RedHat subscriptions for RHOS-backed FUEL*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Now Fuel 3.2 security settings allow RHEL download for RedHat personnel, 
+  if they download RHEL distribution media from internal RedHat network. 
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst]: Added security settings that enable RedHat personnel to download the RHEL distribution media through internal RedHat network.

This is a feature, not an issue.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 9]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+  Now Fuel 3.2 security settings allow RHEL download for RedHat personnel, 
+  if they download RHEL distribution media from internal RedHat network. 
+  
+*PRD-1383 Nodes of OpenStack don't boot up if boot order of disks changed.*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  New partition manager was implemented to eliminate multiple issues, discovered 
+  on multiple hard drive configurations. GRUB no more fails node boot when disk 
+  size exceeds 4 Gb. GRUB no more fails node boot in case of several hard drives
+  connected to the target node.
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst]: Implemented a new partition manager that helps to eliminate issues with many hard drive configurations.
The fixed issues include:
- GRUB fails to boot a node when disk size exceeds 4 GB. (it's GB for Gigabyte and Gb for Gigabit)
- GRUB fails to boot a boot if a few hard drives are connected to the target node

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 10]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+  New partition manager was implemented to eliminate multiple issues, discovered 
+  on multiple hard drive configurations. GRUB no more fails node boot when disk 
+  size exceeds 4 Gb. GRUB no more fails node boot in case of several hard drives
+  connected to the target node.
+  
+*PRD-1476 Fuel can not install master node with disks larger than 3TB*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Now Fuel 3.2 master node can be installed to the disk drive greater than 2.2Tb
+  GPT is used as default partition type.
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst:619]: Added support for Fuel Master Node installation on disk drives that have capacities that exceed 2.2 Tb.
By default, GUID Partition Table is used.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 11]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+*PRD-1476 Fuel can not install master node with disks larger than 3TB*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Now Fuel 3.2 master node can be installed to the disk drive greater than 2.2Tb
+  GPT is used as default partition type.
+  
+*PRD-1498 quantum net unreachable long time (5-6 minutes)* **(M/A::marked as unresolved)**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  L3 agent migration speed time was decreased in order to get Neutron network
+  accessible in lesser time after neutron controller node failure in HA mode.
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst:624]: Decreased the L3 agent migration speed time. In HA mode, this speeds up the `neutron` controller node recovery after failure.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 12]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+  
+*PRD-1498 quantum net unreachable long time (5-6 minutes)* **(M/A::marked as unresolved)**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  L3 agent migration speed time was decreased in order to get Neutron network
+  accessible in lesser time after neutron controller node failure in HA mode.
+  
+*PRD-1499 FuelWeb 3.0.1 Missing keystone_ec2_url configuration option in /etc/nova/nova.conf*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Fuel 3.2 correctly sets keystone_ec2_url parameter to /etc/nova/nova.conf
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst:630]: Fixed the issue with incorrect `keystone_ec2_url` parameter setting in `/etc/nova/nova.conf`

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 13]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+  
+*PRD-1499 FuelWeb 3.0.1 Missing keystone_ec2_url configuration option in /etc/nova/nova.conf*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Fuel 3.2 correctly sets keystone_ec2_url parameter to /etc/nova/nova.conf
+  
+*PRD-1620 Debug logging option enabled out of box in "/etc/openstack-dashboard/local_settings"*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Debug mode for OpenStack dashboard is turned off by default in Fuel 3.2
+  No more unnecessary tons of access logs appear on the controller nodes.
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst:635]: Changed the default setting of debug mode for the OpenStack dashboard to off. 
This eliminates the excessive log messages generation on controller nodes.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 14]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+  
+*PRD-1620 Debug logging option enabled out of box in "/etc/openstack-dashboard/local_settings"*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Debug mode for OpenStack dashboard is turned off by default in Fuel 3.2
+  No more unnecessary tons of access logs appear on the controller nodes.
+  
+*PRD-1716 Default gateway is not defined on slaves for certain configuration*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Now default gateway is correctly set for all configurations.
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst:640]: remove 'Now'

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 15]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+*PRD-1716 Default gateway is not defined on slaves for certain configuration*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Now default gateway is correctly set for all configurations.
+  
+*PRD-1732 Can not add security group to instance with Horizon dashboard*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  OpenStack dashboard issue. Due to input validation bug Horizon incorrectly 
+  allows usage of underscore characters in security group names.
+  This bug is fixed in the Havana OpenStack release.
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst:644]: Fixed the OpenStack dashboard issue. Due to the input validation bug, Horizon incorrectly enables 
the usage of underscores in names of security groups. This issue is fixed in the Havana OpenStack release.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 16]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+*PRD-1732 Can not add security group to instance with Horizon dashboard*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  OpenStack dashboard issue. Due to input validation bug Horizon incorrectly 
+  allows usage of underscore characters in security group names.
+  This bug is fixed in the Havana OpenStack release.
+  
+*PRD-2076** *kvm-qemu crashes due to incorrect disk cache mode*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Fuel 3.2 sets KVM disk cache mode properly on bare-metal nodes.
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst:649]: Fixed the issue when Fuel incorrectly sets cache mode of KVM disks on bare-metal nodes.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 17]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+  This bug is fixed in the Havana OpenStack release.
+  
+*PRD-2076** *kvm-qemu crashes due to incorrect disk cache mode*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Fuel 3.2 sets KVM disk cache mode properly on bare-metal nodes.
+  
+*PRD-2241 Only 10 Gb for / partition* **(M/A::marked as unresolved)**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Default root partition size for nodes was increased in Fuel 3.2
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst:659]: Increased the default root partition size for (compute?) nodes.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 18]
```diff
@@ -475,3 +475,103 @@ Contacting Support
...
+  Fuel 3.2 sets KVM disk cache mode properly on bare-metal nodes.
+  
+*PRD-2241 Only 10 Gb for / partition* **(M/A::marked as unresolved)**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Default root partition size for nodes was increased in Fuel 3.2
+  
+*PRD-2263 Smoke test fails at "Create new security group"*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Smoke tests in Fuel 3.2 UI were fixed.
+
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst]: Fixed smoke tests in Fuel UI.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 19]
```diff
@@ -475,3 +475,103 @@ Contacting Support
 You can contact support online, through E-mail, or by phone.
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst:500]: General comments:
- I rephrased all sentences, so they start with a verb, such as "Fixed", "Added", "Increased", etc. This makes them more descriptive, especially considering the fact that each description contains only 1 - 3 sentences. Example: http://www.vmware.com/support/fusion5/doc/fusion-503-release-notes.html#resolvedissues
- Do not mention "Fuel 3.2" in descriptions. It's clear that we write about issues resolved in Fuel 3.2.
- Some of the issues sound more like enhancements.

File: pages/release-notes/v3-2-grizzly-full.rst [hunk 20]
```diff
@@ -475,3 +475,103 @@ Contacting Support
 You can contact support online, through E-mail, or by phone. 
 Instructions on how to use any of these contact options can be found 
 here: https://mirantis.zendesk.com/home.
+
+
+Issues fixed in this release
+----------------------------
```
Comment (svekars) [inline pages/release-notes/v3-2-grizzly-full.rst:506]: ## Resolved Issues

File: pages/release-notes/v3-2-grizzly-full.rst (hunk 2 above)
Comment (mattymo) [inline pages/release-notes/v3-2-grizzly-full.rst:518]: It's more like: Short DNS name resolution now works on the Management networking, satisfying a technical requirement in RabbitMQ

File: pages/release-notes/v3-2-grizzly-full.rst (hunk 3 above)
Comment (mattymo) [inline pages/release-notes/v3-2-grizzly-full.rst:525]: CentOS not Centos
Red Hat not RedHat

File: pages/release-notes/v3-2-grizzly-full.rst (hunk 10 above)
Comment (mattymo) [inline pages/release-notes/v3-2-grizzly-full.rst:619]: TB not Tb

File: pages/release-notes/v3-2-grizzly-full.rst (hunk 13 above)
Comment (mattymo) [inline pages/release-notes/v3-2-grizzly-full.rst:635]: This also reduces the log size on the Fuel Master as well.
Debug mode can be enabled in Environment Settings before deploying.

File: pages/release-notes/v3-2-grizzly-full.rst (hunk 15 above)
Comment (mattymo) [inline pages/release-notes/v3-2-grizzly-full.rst:644]: Did we backport this bugfix for Grizzly? We're not on Havana, so we must have backported it or it was backported to Grizzly Stable by the upstream maintainers.

Also not allowing underscore is quite weird

**Comment by mihgen** [comment]
Sveta, please work on this after merge. Miroslav, for your future contrubutions - please pay an attention for the audience who will be using these docs. This is user documentation, not development - it means that we should avoid references to source code, and be more descriptive, with the language user would understand. 
"This failure happens by design" - there is no such a thing as failure by design. You would better say something like
"Test is expected to fail unless valid Murano image is uploaded to OpenStack and properly registered. Please follow ... section of documentation for detailed instructions."
-  Murano (Windows virtual machine support in OpenStack) tests failure.
  -  This failure happens by design - manual test preparations from user side
  -  required to run Murano tests.
  -  These steps may be found at https://github.com/Mirantis/fuel-docs/pull/29

**Pull request merged**